tfhe-zk-pok/benches/pke_verify.rs

```diff
@@ -0,0 +1,57 @@
+use criterion::{criterion_group, criterion_main, Criterion};
```

Comment:
can't those be in the same file ? see the integer bench for pke zk

Reply:
But if this is not needed I can easily merge the files

Reply:
you can filter with criterion normally